java/src/jmri/jmrit/decoderdefn/IdentifyDecoder.java

```diff
@@ -308,8 +309,7 @@ public boolean test5(int value) {
         case HORNBY:
             if (modelID == 254) { // HN7000
                 productIDhigh = value;
-                statusUpdate("Read decoder ID CV 49");
-                readCV("49");
+                productID = (productIDhigh << 8) | productIDlow;
                 return false;
```

Comment:
Should this be a "return true"?  The code has finished forming the productID, and hasn't issued another read.